ext/native/thin3d/thin3d_d3d11.cpp

```diff
@@ -1396,15 +1399,17 @@ bool D3D11DrawContext::CopyFramebufferToMemorySync(Framebuffer *src, int channel
 	D3D11_BOX srcBox{ (UINT)bx, (UINT)by, 0, (UINT)(bx + bw), (UINT)(by + bh), 1 };
 	switch (channelBits) {
 	case FB_COLOR_BIT:
-		context_->CopySubresourceRegion(packTex, 0, bx, by, 0, fb->colorTex, 0, &srcBox);
+		context_->CopySubresourceRegion(packTex, 0, bx, by, 0, fb ? fb->colorTex : bbRenderTargetTex_, 0, &srcBox);
 		break;
 	case FB_DEPTH_BIT:
 	case FB_STENCIL_BIT:
 		// For depth/stencil buffers, we can't reliably copy subrectangles, so just copy the whole resource.
-		context_->CopyResource(packTex, fb->colorTex);
+		assert(fb);  // We haven't got a texture for the backbuffer depth. Could make one but I don't think we need one.
```

Comment:
Agreed - may want to double check it doesn't segfault when using non-buffered rendering (even if depth/stencil isn't available) in GE debugger.

-[Unknown]

Reply:
It indeed did, and I changed the error checking.